roles/users/README.rst

```diff
@@ -0,0 +1,87 @@
+Users
+=====
+
+This role is used to manage user accounts on a node.  It is capable of setting up users
+with and without sudo access, but does not currently have the ability to remove users.
+In either your group_vars or host_vars files you can define two variables for this role
+to use: ``managed_users`` and ``managed_admin_users``.  The ``managed_users`` variable
+will create users without sudo access while users in the ``managed_admin_users`` list
+will be granted sudo access. Sudo access is granted by adding the ``managed_admin_users`` to
```

Comment:
Maybe say briefly what the ... playbook?... does in total; I assume it's "add the user, create a home dir, add .ssh/authorized_keys, etc.  and maybe note which tasks are accomplished by the "user" tag and which are in the "pubkeys" tag